install-devenv.sh

```diff
@@ -180,7 +180,13 @@ main() {
 
   show "$devenv_python" -m venv --clear "$devenv_venv"
 
-  show "$devenv_venv"/bin/pip install "git+$SNTY_DEVENV_REPO@$SNTY_DEVENV_BRANCH"
+  if [[ "${CI:-}" ]]; then
+    show "$devenv_venv"/bin/pip install "git+$SNTY_DEVENV_REPO@$SNTY_DEVENV_BRANCH"
+  else
+    show "$devenv_venv"/bin/pip install \
+      --index-url https://pypi.devinfra.sentry.io/simple \
+      "sentry-devenv==${SNTY_DEVENV_VERSION}"
```

Comment:
could even just leave this unpinned so that our *released* version is what gets pulled in

Reply:
:hmmyes: